webapi/webapi-contactsmanager-w3c-tests/contactsmanager/ContactsManager_save_Contact_fullset.html

```diff
@@ -68,7 +68,26 @@
     preferred: true,
     value: 'msn:kangx.xu@intel.com'
   });
-  var data = new Date(1987,10,27);
+  Date.prototype.format = function(format) {
```

Comment:
@zqzhang , @LaneRuan is about to leave. I will take this issue.
You're right, the best way to compare two dates are by timestamp, which is get by `getTime()`. So all date objects should convert to timestamp when they compare to each other.
For this issue, it's easy to convert Date object to timestamp in Javascript. But the time get by Java code is not a real Javascript Date object. That part of code should be changed. But it's far from our ownership. Could you contact the right person to fix this?